DataFormats/Math/interface/ExtVec.h

```diff
@@ -3,21 +3,47 @@
 
 #include<type_traits>
 
-typedef float  __attribute__( ( vector_size(  8 ) ) ) float32x2_t;
-typedef float  __attribute__( ( vector_size( 16 ) ) ) float32x4_t;
-typedef float  __attribute__( ( vector_size( 32 ) ) ) float32x8_t;
-typedef double __attribute__( ( vector_size( 16 ) ) ) float64x2_t;
-typedef double __attribute__( ( vector_size( 32 ) ) ) float64x4_t;
-typedef double __attribute__( ( vector_size( 64 ) ) ) float64x8_t;
+#ifdef __clang__
```

Comment:
Because we have `DataFormats/Math/src/ExtVec.cc`, which is always compiled. That's the one trigger compilation issues. It contains a number of operators for printing things like `Vec2D`. One could disable those, but there is no top level header file which selects `USE_EXTVECT/USE_SSEVECT`. Current decision is done in `DataFormats/Math/interface/SIMDVec.h`, but you cannot include it into  `DataFormats/Math/src/ExtVec.cc` as that would create cyclic dependency. Someone would need to split `SIMDVec.h`.